containers/compilers/osv/java-compiler/java-main-caller/src/main/java/com/emc/wrapper/UDPBootstrap.java

```diff
@@ -1,160 +0,0 @@
-package com.emc.wrapper;
```

Comment:
Since the modified capstan now integrates java a bit differently, the initial idea was to remove the current version completely. We discussed yesterday with @miha-plesko about this PR and decided to abandon it an split into several smaller PRs, incrementally introducing the new capabilities for OSv.

One of the things we decided was to stick with current support for Java and add just new support for Node and "native". This will facilitate discussion about the proposed benefits, as well as simplify the review process.